administration/src/bp-modules/applications/ApplicationCard.tsx

```diff
@@ -81,7 +81,12 @@ const DeleteDialog = (p: {
   return (
     <Dialog open={p.isOpen} aria-describedby='alert-dialog-description'>
       <DialogContent id='alert-dialog-description'>
-        <Stack direction='row' gap={2} alignItems='center'>
+        <Stack
+          direction='row'
+          sx={{
+            gap: 2,
+            alignItems: 'center',
+          }}>
```

Comment:
This would fit on one line
```suggestion
        <Stack direction='row' sx={{ gap: 2, alignItems: 'center' }}>
```